{'project_id': '3130', 'year': '1984', 'x': 77340, 'y': 851520, 'plot_id': '17321', 'strata': '41'}
Pandas(Index=0, project_id='3130', plot_id='17321', x=77340, y=851520)
called process_plot
Pandas(Index=0, project_id='3130', plot_id='17321', x=77340, y=851520)
17321
Directory '/efs/timesync/1984/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_17321_3130.p'.
.....................Failed qa: 1984-04-27T16:17:23.1120310Z
Failed qa: 1984-05-20T16:24:23.405047Z
..............Failed qa: 1984-06-21T16:25:01.200082Z
Failed qa: 1984-06-30T16:18:44.6760500Z
Failed qa: 1984-07-07T16:25:11.566069Z
.......Failed qa: 1984-09-02T16:20:18.4110560Z
Failed qa: 1984-09-09T16:26:42.976047Z
.................................................Data has been pickled and saved to '/efs/timesync/1984/audit/1984_17321_3130_qa_rejects.p'.
Success tracking goes here
